docs/manager.md

```diff
@@ -1,7 +1,7 @@
 # Managers/Traders
 
 ## Overview
-The future owners of protocol on whose success the experiment of running a decentralized fund management protocol can be made possible in a truly trustless and non custodial way. The protocol has been keenly designed to incentivize traders who provide their expertise on crypto markets to benefit both the users of protocol.
+The future owners of protocol on whose success the experiment of running a decentralized fund management protocol can be made possible in a truly trustless and non custodial way. The protocol has been keenly designed to incentivize traders who provide their expertise in DeFi to benefit both the users of protocol.
```

Comment:
The goal of IVN is to have the future owners of protocol be the Managers. Therefore, the success of running a decentralized fund management protocol is reliant on them. The protocol has been keenly designed to incentivize traders to become Managers by providing and using their expertise in DeFi to benefit all users of protocol.